lib/core/src/client/preview/start.js

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import deprecate from 'util-deprecate';
 import AnsiToHtml from 'ansi-to-html';
+import escape from 'lodash/escape';
```

Comment:
Oops. This is left from my previous attempt at tackling the problem. Gonna remove it in a sec.